src/Text/Pandoc/Readers/MediaWiki.hs

```diff
@@ -667,7 +667,7 @@ internalLink = try $ do
   sym "]]"
   -- see #8525:
   linktrail <- B.text <$> manyChar (satisfy (\c -> isLetter c && not (isCJK c)))
-  let link = B.link (addUnderscores pagename) "wikilink" (label <> linktrail)
+  let link = B.linkWith (mempty, ["wikilink"], mempty) (addUnderscores pagename) (stringify label) (label <> linktrail)
```

Comment:
Can you comment on the reason for putting `stringify label` in the title?

Reply:
This (approximately) aligns with what mediawiki does, and I tossed in this change since I freed up the `title` on the mediawiki reader. I don't mind leaving it out of this change as orthogonal.

Reply:
I'm okay leaving it in if it's what mediawiki does.

Reply:
OK, this is ready to go in now then!